src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -65,6 +72,11 @@ public AppealController(AppealService appealService) {
 
             return ResponseEntity.created(location).body(id);
 
+        } catch (AppealReasonException appealReasonException) {
+            LOGGER.error(appealReasonException.getMessage() +
+                " for company number {}, penalty reference {} and user id {}",
+                    companyId, penaltyReference, userId, appealReasonException);
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
```

Comment:
Exceptions and error logging should be used for exceptional circumstances. A boolean would remove the need for the custom exception.

```
if(!appealReasonValidator.isValid(appeal.getReason()) {
    LOGGER.info(...);
    return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
}
```

Reply:
That would be more standard with common validator patterns. However I am still going to continue to argue that this should be handled by the pre-existing Spring validation, rather than creating a parallel system to do it.